src/scss/components/hero-banner/_navigation-white.scss

```diff
@@ -79,7 +79,7 @@
           > .su-multi-menu__item--parent {
             > .su-multi-menu__link {
               &::after {
-                border-right-color: $su-color-white;
+                border-right-color: transparent;
```

Comment:
Given your change in `_main-nav.scss`, I don't think this `border-right-color` is needed anymore. Do you want to remove it?

Reply:
Yes. Done.